jax/_src/pallas/pallas_call.py

```diff
@@ -1072,14 +1073,31 @@ def wrapped(*args):
                           for a in flat_args)
     flat_out_avals = tuple(jax_core.ShapedArray(v.shape, v.dtype)
                            for v in flat_out_shapes)
+
+    kernel_fun_sig = api_util.fun_signature(kernel)
+    arg_names = None
+    kernel_src_info = "<unknown>"
+    if kernel_fun_sig:
+      kernel_debug_info = api_util.debug_info(
```

Comment:
Wdyt about using `api_util._arg_names` directly (and making it public) instead of going through `debug_info` which we don't actually need?

Reply:
I tried various things here. I think that JAX's arg name machinery deserves to be cleaned up. I would rather not open up more APIs at this point.